src/mixins/auth-api-mixin.ts

```diff
@@ -0,0 +1,27 @@
+// Libraries
+import axios from '@/utils/axios-auth'
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+
+/**
+ * Mixin that provides integration with the Auth API.
+ */
+@Component({})
+export default class AuthApiMixin extends Vue {
+  @Getter getBusinessId!: string
+
+  /**
+   * Fetches authorizations.
+   * @param businessIdentifier the business identifier (eg, BC1219948)
+   * @returns a promise to return the authorizations object
+   */
+  async fetchAuthorizations (businessIdentifier: string): Promise<any> {
+    if (!businessIdentifier) throw new Error('Invalid parameter \'businessIdentifier\'')
+
+    const url = `entities/${businessIdentifier}/authorizations`
+    const authUrl = sessionStorage.getItem('AUTH_API_URL')
+    const config = { baseURL: authUrl }
```

Comment:
If this gets complicated in unit testing, we can just make `authURL` part of `url` and not need to specify baseURL.

Reply:
kk